doc/glossary.rst

```diff
@@ -174,6 +174,11 @@ general neuroimaging concepts. If you think a term is missing, please consider
         data in a common space for statistical analysis).
         See :ref:`ch_morph` for more details.
 
+    noise covariance
+        Noise covariance is an estimate of the correlation between each pair of
```

Comment:
correlation != covariance

How about something like this:

```
Noise covariance is a matrix that contains the covariance between data channels.
It is used to spatially whiten data for example for source imaging. It is especially
useful to work with different sensor types. It is in practice estimated from
baseline periods or empty room measurements.
```

Reply:
I think it would be helpful to mention what is the shape of this matrix. It helps understand what it captures.

I wouldn't mention whitening because that's throwing in another term that many beginners don't know about.

Reply:
ok for me.